src/index.js

```diff
@@ -0,0 +1,108 @@
+import './styles.scss';
+import celsiusToFahrenheit from './functions';
+import showError from './dom';
+
+// get the elements
+const weatherIcon = document.querySelector('.weather-icon');
+const weatherDescription = document.querySelector('.description');
+const tempCelsius = document.querySelector('.temp-celsius');
+const tempMin = document.getElementById('temMin');
+const tempMax = document.getElementById('temMax');
+const location = document.querySelector('.location');
+const todayDate = document.getElementById('todayDate');
+const inputCity = document.getElementById('address-input');
+
+// app data
+const weather = {};
+weather.temperature = {
+  unit: false,
+};
+
+const KELVIN = 273;
+const KEY = '15b72f8181c849f71bb8b90b88730574';
+
+const displayWeather = () => {
```

Comment:
- [ ] Since this function is a DOM-related, maybe we should move it to `dom.js`?